vms/platformvm/state/camino_test.go

```diff
@@ -162,6 +268,98 @@ func TestSyncGenesis(t *testing.T) {
 	}
 }
 
+func defaultConfig(lockModeBondDeposit bool, nodeID ids.NodeID, deposit bool) root_genesis.Config {
+	depositOffers := []genesis.DepositOffer{
+		{
+			OfferID:                 id,
+			InterestRateNominator:   1,
+			Start:                   2,
+			End:                     3,
+			MinAmount:               4,
+			MinDuration:             5,
+			MaxDuration:             6,
+			UnlockPeriodDuration:    7,
+			NoRewardsPeriodDuration: 2,
+			Flags:                   9,
+		},
+	}
+
+	if deposit {
+		id0, _ = depositOffers[0].ID()
+	}
+
+	caminoAllocations := make([]root_genesis.CaminoAllocation, 1)
+	allocations := make([]root_genesis.Allocation, 1)
+
+	platformAllocations := []root_genesis.PlatformAllocation{
+		{
+			Amount:            minStake,
+			NodeID:            nodeID,
+			ValidatorDuration: defaultMinValidatorStake,
+			DepositDuration:   5,
+			TimestampOffset:   10,
+			DepositOfferID:    id0,
+			Memo:              "",
+		},
+	}
+
+	if lockModeBondDeposit {
+		caminoAllocations = []root_genesis.CaminoAllocation{
+			{
+				ETHAddr:  ids.GenerateTestShortID(),
+				AVAXAddr: ids.GenerateTestShortID(),
+				XAmount:  10 * defaultMinValidatorStake,
+				AddressStates: root_genesis.AddressStates{
+					ConsortiumMember: true,
+					KYCVerified:      true,
+				},
+				PlatformAllocations: platformAllocations,
+			},
+		}
+	} else {
+		allocations = []root_genesis.Allocation{
+			{
+				ETHAddr:       initialAdmin,
+				AVAXAddr:      initialAdmin,
+				InitialAmount: minStake,
+				UnlockSchedule: []root_genesis.LockedAmount{
+					{
+						Amount:   1,
+						Locktime: 0,
+					},
+				},
+			},
+		}
+	}
+
+	config := root_genesis.Config{
+		NetworkID:                  0,
+		Allocations:                allocations,
+		StartTime:                  10,
+		InitialStakeDuration:       10,
```

Comment:
make much bigger value
10 seconds could result in zero reward in avax-mode even if there should be some validator rewards